docs/production/native_cloud.md

```diff
@@ -1,26 +1,36 @@
 # Mage and Native Cloud Environments
+
 Just like a traditional notebook, Mage supports execution in native cloud environments. Below are guides on how to integrate with native cloud resources.
-- [Amazon EC2](#amazon-ec2)
 
-Below are guides on how to use these launch scripts
-## Amazon EC2
-Mage can be run in an Amazon EC2 instance, either by [manually setting up a connection](#manual-connection) or by using the [Quick Connection](#quick-connection) script provided.
+-   [Amazon EC2](#amazon-ec2)
+
+# Amazon EC2
+
+Mage can be run in an Amazon EC2 instance, either by
+
+-   using SSH to port forward localhost requests to your EC2 instance
+-   opening port 6789 on your EC2 instance for access
 
-### Prerequistes
-The only prerequisite is that your EC2 instance must have `python3` installed.
-### Quick Connection
+## Prerequistes
+
+Your EC2 instance must have `python3` installed.
```

Comment:
We support python 3.7 and above. I remember the tests failed in python 3.6. You can double check.